### DISCUSSION TITLE
CMAF/dash so slow to preload

### DISCUSSION THREAD

**Discussion by reiluke** [ant-media/Ant-Media-Server#5619] [General]
Hi Team,

CMAF/dash looks like will fit with our application with lots of viewers, but is very slow at start compared to webrtc.

At start, event "waiting" and "play" is always called, audio stops and play you need to wait a couple of minutes till it stablize, this does not happen with webrtc.

We tried a lot of settings but its the same, if this is normal is there anyway to show a loading screen until good for smooth playback.

Thank you

**Comment by USAMAWIZARD** [comment]
Hi @reiluke , 
 
I tried play with Dash its stable and working fine for me can you please let me know which version of Ant Media Server are you using. Please let me know if you have any reproduce scenario for the issue

**Comment by reiluke** [comment]
Hi, we are using latest 2.6.4 
sample video

https://drive.google.com/file/d/1koeSmxsBg1jERxGsRYQU6YWbLIQ2H_uy/view?usp=sharing

highspeed internet 1gb u/d, no problems with webrtc even if internet is slow

thanks!

**Reply by yashtandon113** [reply]
HI @reiluke 
It is a known issue and we already created the GitHub issue for this problem.
https://github.com/ant-media/Ant-Media-Server/issues/5628

It would be good if you can comment on the issue to increase its priority.

**Reply by reiluke** [reply]
Thank you, btw we have tried basic dashjs videojs using mpd generated from antmedia and it works fine. https://videojs.github.io/videojs-contrib-dash/ so it must in the antmedia player
